pkg/otlp/metrics/metrics_translator.go

```diff
@@ -575,6 +577,7 @@ func mapHistogramRuntimeMetricWithAttributes(md pmetric.Metric, metricsArray pme
 
 // MapMetrics maps OTLP metrics into the DataDog format
 func (t *Translator) MapMetrics(ctx context.Context, md pmetric.Metrics, consumer Consumer) error {
+	t.hasRuntimeMetrics = false
```

Comment:
Where else is it used? If it needs to be reset on every call to `MapMetrics`, why not just make it a return value>

Reply:
As far as I know, only once in the Agent and once in the Collector, but I wasn't sure if it's used anywhere else.